include/core/ib_particle.h

```diff
@@ -85,47 +85,69 @@ class IBParticle
 
   // This class defines values related to a particle used in the sharp interface
   // IB. Each particle defined will have these value used in the solver.
-  Point<dim>     position;
-  Point<dim>     last_position;
-  Tensor<1, dim> forces;
-  Tensor<1, dim> last_forces;
-  unsigned int   particle_id;
+  Point<dim>              position;
+  std::vector<Point<dim>> last_position;
+  Tensor<1, dim>          forces;
+  Tensor<1, dim>          last_forces;
+  unsigned int            particle_id;
 
   Tensor<1, 3> torques;
+  Tensor<1, 3> last_torques;
```

Comment:
```suggestion
  Tensor<1, 3> torque;
  Tensor<1, 3> last_torque;
```
These should be single no?